### PR TITLE
[TIMOB-18744] Android: setting touchEnabled true by default

### PR DESCRIPTION
**Pull request by ashcoding**
https://jira.appcelerator.org/browse/TIMOB-18744

Test Code:-

``` javascript
var win = Titanium.UI.createWindow({
    title : 'Tab 1',
    backgroundColor : '#fff',
    layout:"vertical"
});

// Create a Button.
var save = Ti.UI.createButton({
    title : 'save',
    height : Ti.UI.SIZE,
    width : Ti.UI.SIZE,
    top : 30,
});



// Listen for click events.
save.addEventListener('click', function(e) {
    alert('touchEnabled = ' + typeof (e.source.touchEnabled) + ' ; value = ' + e.source.touchEnabled);
});

// Create a Label.
var testlabel = Ti.UI.createLabel({
    text : 'testlabel',
    color : '#000',
    font : {
        fontSize : 40
    },
    height : Ti.UI.SIZE,
    width : Ti.UI.SIZE,
    top : 50,

});

// Listen for click events.
testlabel.addEventListener('click', function(e) {
    alert('touchEnabled = ' + typeof (e.source.touchEnabled) + ' ; value = ' + e.source.touchEnabled);
});

// Add to the parent view.


var view = Ti.UI.createView({
    width : 100,
    height : 50,
    backgroundColor : 'red',
    top:20

});

// Listen for click events.
view.addEventListener('click', function(e) {
    alert('touchEnabled = ' + typeof (e.source.touchEnabled) + ' ; value = ' + e.source.touchEnabled);
});

// Add to the parent view.
win.add(testlabel);
win.add(view);
win.add(save);

win.open();
```

### CODE REVIEW

File: android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java [hunk 1]
```diff
@@ -70,7 +70,7 @@
 	"softKeyboardOnFocus", "transform", "elevation", "touchTestId",
 	"translationX", "translationY", "translationZ",
 	
-	TiC.PROPERTY_TRANSITION_NAME
+	TiC.PROPERTY_TRANSITION_NAME, TiC.PROPERTY_TOUCH_ENABLED
```
Comment (hieupham007) [inline android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java]: "touchEnabled" should already be an accessor, look at line 69

**Comment by hieupham007** [comment]
Code reviewed. Left a comment

File: android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java (hunk 1 above)
Comment (ashcoding) [inline android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java]: Got it.

**Comment by ashcoding** [comment]
Addressed comment.

**Comment by hieupham007** [comment]
Code reviewed and tested. Looks good.

**Pull request merged**